desktop/apps/artwork/components/auction/helpers.coffee

```diff
@@ -1,12 +1,17 @@
 { compact } = require 'underscore'
+Auction = require '../../../../models/auction.coffee'
 
 pluralize = (word, count, irregular = null) ->
   if count is 1
     "#{count} #{word}"
   else
     "#{count} #{irregular or word + 's'}"
 
+liveAuctionUrl = (auction_id, user) ->
+  (new Auction(id: auction_id)).liveAuctionUrl(user)
```

Comment:
I agree -- this seems a little bit unclear, and could also create unwanted, unknown side-effects further down the line should things change. To get a url, one shouldn't have to execute a `constructor`.